tests/test_app_basic_combined.py

```
@@ -0,0 +1,76 @@
+        # Just check that the app has some routes
+        assert hasattr(app.app, "routes")
+        assert len(app.app.routes) > 0
+
+
+class TestAppVIPIntegration:
+    """Test VIP module integration."""
+
+    def test_app_vip_integration_success(self):
+        """Verify that VIP routes register successfully when the module is enabled."""
+        # Since VIP module is currently working, test that it's properly integrated
+        fastapi_app = getattr(app, "app")
```

Ruff (0.14.1)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)